docs/io-handbook/technology-radar.md

```diff
@@ -0,0 +1,134 @@
+- [Security policy as code](https://www.thoughtworks.com/radar/techniques/security-policy-as-code)
+- [Simplest possible feature toggle](https://www.thoughtworks.com/radar/techniques/simplest-possible-feature-toggle)
+- [Secrets as a service](https://www.thoughtworks.com/radar/techniques/secrets-as-a-service)
+- [Tailored service templates](https://www.thoughtworks.com/radar/techniques/tailored-service-templates)
+- [Testing using real device](https://www.thoughtworks.com/radar/techniques/testing-using-real-device)
+- [Use "remote native" processes and approaches](https://www.thoughtworks.com/radar/techniques/use-remote-native-processes-and-approaches)
+- [Serverless architecture](https://www.thoughtworks.com/radar/techniques/serverless-architecture)
+
+#### Trial
+
+- [Observability as code](https://www.thoughtworks.com/radar/techniques/observability-as-code)
+- [Platform engineering product teams](https://www.thoughtworks.com/radar/techniques/platform-engineering-product-teams)
+- [Pipelines for infrastructure as code](https://www.thoughtworks.com/radar/techniques/pipelines-for-infrastructure-as-code)
+
+#### Assess
+
+- [JAMstack](https://www.thoughtworks.com/radar/techniques/jamstack)
+- [Polyglot programming](https://www.thoughtworks.com/radar/techniques/polyglot-programming)
+- [The three Rs of security](https://www.thoughtworks.com/radar/techniques/the-three-rs-of-security)
+- [Wardley mapping](https://www.thoughtworks.com/radar/techniques/wardley-mapping)
+- [Team Topologies](https://teamtopologies.com/key-concepts)
+
+#### Hold
+
+- [Log aggregation for business analytics](https://www.thoughtworks.com/radar/techniques?blipid=202005090)
+- [Microservice envy](https://www.thoughtworks.com/radar/techniques/microservice-envy)
+- [Templating in YAML](https://www.thoughtworks.com/radar/techniques/templating-in-yaml)
+
+## Tools
+
+#### Adopt
+
+- [Airflow](https://www.thoughtworks.com/radar/tools/airflow)
+- [Dependabot](https://www.thoughtworks.com/radar/tools/dependabot)
+- [ESLint](https://www.thoughtworks.com/radar/tools/eslint)
+- [Parcel](https://www.thoughtworks.com/radar/tools/parcel)
+- [Prettier](https://www.thoughtworks.com/radar/tools/prettier)
+- [Snyk](https://www.thoughtworks.com/radar/tools/snyk)
+- [Terraform](https://www.thoughtworks.com/radar/tools/terraform)
+- [Terragrunt](https://www.thoughtworks.com/radar/tools/terragrunt)
+- [Visual Studio Code](https://www.thoughtworks.com/radar/tools/visual-studio-code)
+- [Yarn](https://www.thoughtworks.com/radar/tools/yarn)
+
+#### Trial
+
+- [Headless Chrome for front-end test](https://www.thoughtworks.com/radar/tools/headless-chrome-for-front-end-test)
+- [tfsec](https://www.thoughtworks.com/radar/tools/tfsec)
+- [Terratest](https://www.thoughtworks.com/radar/tools/terratest)
+- [Visual Studio Live Share](https://www.thoughtworks.com/radar/tools/visual-studio-live-share)
```

Comment:
Anecdotal: 
Live Share is very stable and reliable, it works well and I encourage to use it as default during pairing session.